### PR TITLE
Deploy Jazzy site automatically from Travis CI

### PR DESCRIPTION
**Pull request by aaronsky**
Enables this using a GitHub Personal Access Token from my account. We can discuss the particulars of this strategy in review.

### CODE REVIEW

**Comment by simonpierreroy** [comment]
LGTM! When should we change the Source for GitHub Pages to "master branch / docs folder"?

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Jaeger-Swift is a distributed tracing client for Uber's [Jaeger](https://www.jae
 
 ## Installation
 
-This section has quick setup steps for getting a simple proof of concept up and running. For a more comprehensive understanding of this project, Check out our [API reference](). A complete Opentracing solution consists of a client, an agent and a collector. The following steps will show you how to configure this project to report to a locally hosted Jaeger collector.
+This section has quick setup steps for getting a simple proof of concept up and running. For a more comprehensive understanding of this project, Check out our [API reference](https://wayfair.github.io/jaeger-swift/docsets/jaeger-swift.xml). A complete Opentracing solution consists of a client, an agent and a collector. The following steps will show you how to configure this project to report to a locally hosted Jaeger collector.
```
Comment (aaronsky) [inline README.md]: I'm sorry, I think I misunderstood what you meant by linking to the docset. The reference URL should link to the site itself, at `https://wayfair.github.io/jaeger-swift`

File: README.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ class ViewController: UIViewController {
...
+In the sinppet shown above, we use the pre-built `JaegerCoreDataClient` which implements the `Agent` protocol and `JSONSender`, which, implements the `SpanSender` protocol. The sender accepts an array of type `SpanConvertible` which is a customizable representation of a span. Now, if you need to implement a strategy to report to a different collector like Zipkin, you can implement the `SpanConvertible` protocol to convert a `Span` into the appropriate format accepted by Zipkin. If you need a reference, we have done the same with `JaegerSpan`. The `Agent` and `SpanSender` being protocols are also customizable. If you need more information on this topic, you can refer our [API Documentation](https://wayfair.github.io/jaeger-swift).
```
Comment (jdhealy) [inline README.md]: Typo: `sinppet` → `snippet`

**Comment by aaronsky** [comment]
@jdhealy @simonpierreroy1 This is ready to go

**Review by jdhealy — APPROVED** [review]

**Pull request merged**